src/i18n/TextCollection/i18nTextCollector.php

```diff
@@ -378,8 +378,8 @@ protected function mergeWithExisting($entitiesByModule)
             // Merge
             if ($existingMessages) {
                 $entitiesByModule[$module] = array_merge(
-                    $existingMessages,
-                    $messages
+                    $messages,
```

Comment:
If we do it this way, any existing messages with new defaults won't be updated, won't they?